client/my-sites/domains/domain-management/list/bulk-site-domains.tsx

```diff
@@ -60,9 +61,7 @@ export default function BulkSiteDomains( props: BulkSiteDomainsProps ) {
 	const isInSupportSession = Boolean( useSelector( isSupportSession ) );
 
 	const hasNonWpcomDomains = useMemo( () => {
-		return (
-			filterDomainsByOwner( filterOutWpcomDomains( data?.domains ?? [] ), undefined ).length > 0
-		);
```

Comment:
What motivated this change?

Reply:
When I was deleting the unused code I realised that the type signature for the `filterDomainsByOwner` type was incorrect. So I went to fix it. And when doing that I discovered that `filterDomainsByOwner` doesn't even do anything, so I didn't need to fix the type issue, I could just remove it.

Notice we were calling `filterDomainsByOwner` with `undefined` as the filter type. The `undefined` means "filter nothing"

Reply:
Lol. Wtf... Haha.

Reply:
And the weirdest thing is that I introduced it, including the `undefined` since there is no way to filter in the new domains table. Why are we accepting the `undefined` param, then? It doesn't make much sense in my head.

Reply:
I think it just came over as part of copy-pasting the old filtering code. The old page had a drop down for choosing how to filter owner, which is a feature we no longer have.
You probably figured that we were going to implement filtering eventually and so that `undefined` would be replaced by the current state of the filtering controls. Alas that never came to be 😮‍💨